vision-zero

```diff
@@ -131,8 +131,6 @@ def replicateDb(args):
     # fmt: off
     replicate_runner_command = docker_compose_invocation + [
         "-f", "docker-compose.yml", 
-        "-f",
-        "docker-compose-ram-disk.yml" if args.ram_disk else "docker-compose-docker-volume.yml",
```

Comment:
Should this arg be removed from the arg parser and also from the local dev docs now that this Docker Compose file is removed?

Reply:
Yes! Thank you - great catch! c157b8656eaf421e53eea7a91a7d1b6c22c64f78